.env.development

```diff
@@ -0,0 +1,4 @@
+PORT=3001
```

Comment:
so this is a bit tricky here.

DEVELOPMENT MODE:
`react-scripts start` will set `NODE_ENV=development`, which in turn loads `.env.development` (on top of the default `.env`)

In this mode, we've got 2 servers running, one is react-scripts on `PORT=3001`, the other is express (on port 3000, see the other comment in server.js).

So we serve the static files from localhost:3001, but all client-side fetch() calls should then point to `REACT_APP_API_URL=localhost:3001`, which is where our API is running.

PRODUCTION MODE:
On the other hand, `react-scripts build` will use `NODE_ENV=production`, which will only load `.env`.

In this mode, all is served by our express server, so we only have PORT=3000, and REACT_APP_API_URL is an empty string, which makes the fetch() URL a relative URL, i.e. the requests go to current host. Which is fine, since we've got both API and static files served from the same host.